content/docs/sidebar.json

```diff
@@ -85,14 +85,12 @@
         "source": "project-structure.md"
       },
       {
-        "slug": "data",
-        "label": "Working with data",
-        "source": "user-guide/data.md"
+        "label": "Working with Data",
+        "slug": "data"
       },
       {
-        "slug": "remote-projects",
-        "label": "Working with remote projects and objects",
-        "source": "user-guide/remote-projects.md"
+        "label": "Working with Remote Objects",
+        "slug": "remote-objects"
```

Comment:
Note file rename to match contents/title
and that you don't need to specify `source` when the `slug` implies it
+ personally I prefer `label` before `slug`.

- [ ] Should apply the latter to the whole file (separate PR)